src/styles.css

```
@@ -609,8 +665,27 @@
 		color: var(--muted-foreground);
 	}
 
+	.workspace-document-prose citation {
```

Stylelint (17.14.1)

[error] Unknown type selector "citation" (selector-type-no-unknown)

(selector-type-no-unknown)